Modules/xASL_module_ASL.m

```diff
@@ -843,22 +727,6 @@
     x.Q.BackgroundSuppressionNumberPulses = x.BackgroundSuppressionNumberPulses;
 end
 
-if ~isfield(x.modules.asl, 'bUseBasilQuantification') || isempty(x.modules.asl.bUseBasilQuantification)
```

Comment:
Why is this part not called here to the new function?

Reply:
Parsing of PLDs is in subfunction which is called from the original location.

Setting bUseBasil and bQuantify* is moved to a subfunction that is only called later before quantification. We have discussed why it can't be called twice and agreed to move it before quantification.

Reply:
Sorry but where is xASL_module_ASL_MultiParameterQuantificationOptions?
Either you give me in full detail what you changed and why, or I cannot check it.
I don't know the order by heart, so if you would have just moved parts of the subfunction to new subfunctions and didn't change the order, then it would be easy to check.

Now, for me to really review if you didn't break anything, I need to check in-depth what each part was doing, where it was used, where dependencies are, etc. Because you changed the original functionality, which we didn't agree upon. We only agreed on moving a part of the subfunction to a new subfunction. Not on changing the order of subfunctions or something.

So I'm fine approving this, but then we risk that you moved stuff around that breaks other things and is not properly checked. So I would be more happy if you can do as we agreed: only moving parts of subfunctions.

Afaik the golden rule of commits: changing as little as possible to obtain the new functionality. A larger order change like you did now makes multiple branches etc working at the same time on this function very challenging.